pkg/github/projects.go

```diff
@@ -877,21 +916,53 @@ type projectV2Field struct {
 	UpdatedAt *github.Timestamp `json:"updated_at,omitempty"` // The time when this field was last updated.
 }
 
+type projectV2ItemFieldValue struct {
```

Comment:
small oversight but types in Go start with a capital

```suggestion
type ProjectV2ItemFieldValue struct {
```

we will need to go with a vscode refactor to change this properly, so please don't commit my suggestion 😄